tests/Client.UiTests/Runner.fs

```diff
@@ -0,0 +1,22 @@
+module Client.UiTests.Runner
+
+open canopy.classic
+open canopy.types
+open Expecto
+
+let startBrowser() =
+    //start Chrome // Use this if you want to see your tests in the browser
+    start ChromeHeadless
+    resize (1280, 960)
```

Comment:
this is for debugging, you can just comment out and you'll see browser being open

Reply:
Nono, I meant `resize (1280, 960`. Looks like this is done in order to open the second window with reports on the left part of the screen - but we don't have one so I guess this can be removed.

Reply:
this one sets the size of browser, independent of it being displayed